components/button/button.html

```diff
@@ -0,0 +1,190 @@
+                    industry.
+                </div>
+
+                <div class="example-div">
+                    <button class="btn btn-primary">PRIMARY</button>
+                    <button class="btn btn-secondary">SECONDARY</button>
+                    <button class="btn btn-success">SUCCESS</button>
+                    <button class="btn btn-error">ERROR</button>
+                    <button class="btn btn-dark">DARK</button>
+
+
+                </div>
+
+                <div class="code-snippet"></div>
+                <div class="divider"></div>
+
+                <div class="content-title2"> Basic Buttons</div>
+                <div class="content-subtitle"> Lorem Ipsum is simply dummy text of the printing and typesetting
+                    industry.
+                </div>
+
+                <div class="example-div">
+                    <button class="btn outline-primary">PRIMARY</button>
+                    <button class="btn outline-secondary">SECONDARY</button>
+                    <button class="btn outline-success">SUCCESS</button>
+                    <button class="btn outline-error">ERROR</button>
+                    <button class="btn outline-dark">DARK</button>
+
+                </div>
+
+                <div class="code-snippet"></div>
+                <div class="divider"></div>
+
+                <div class="content-title2"> Link Buttons</div>
+                <div class="content-subtitle"> Lorem Ipsum is simply dummy text of the printing and typesetting
+                    industry.
+                </div>
+
+                <div class="example-div">
+                    <button class="btn btn-primary" href="#">PRIMARY</button>
+                    <button class="btn btn-secondary" href="#">SECONDARY</button>
+
+
+                </div>
+
+                <div class="code-snippet"></div>
+                <div class="divider"></div>
+
+                <div class="content-title2"> Buttons with icon</div>
+                <div class="content-subtitle"> Lorem Ipsum is simply dummy text of the printing and typesetting
+                    industry.
+                </div>
+
+                <div class="example-div">
+                    <button class="btn btn-primary btn-icon">
+                        <box-icon name='cart'></box-icon>Add to cart
+                    </button>
+                    <button class="btn btn-secondary btn-icon">
+                        <box-icon name='send'></box-icon>Send
+                    </button>
+                    <button class="btn btn-error btn-icon">
+                        <box-icon name='error-circle'></box-icon>ERROR
+                    </button>
+                    <button class="btn outline-dark btn-icon">
+                        <box-icon name='search-alt'></box-icon>Search
+                    </button>
+
+                </div>
+
+                <div class="code-snippet"></div>
+                <div class="divider"></div>
+
```

Comment:
A ui Suggestion
![Capture](https://user-images.githubusercontent.com/36101139/155843488-b468a31e-4a27-4424-8481-b449d38bdd31.PNG)

Here the contrast of the yellow button way to low , you can increase the contrast ratio.

Reply:
Noted.